src/System.Diagnostics.Process/tests/RemotelyInvokable.cs

```diff
@@ -87,6 +87,18 @@ public static int ReadLine()
             return SuccessExitCode;
         }
 
+        public static string WriteLineReadLineUapCmd()
```

Comment:
Sorry, usage is here: https://github.com/dotnet/corefx/pull/22413/files#diff-21d130bd09db15e78009186525a10a4dR179